test/hotspot/jtreg/runtime/ClassFile/ClassFileVersionTest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2022, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ *
+ */
+
+/**
+ * @test
+ * @bug 8291360
+ * @summary Test getting the class file version for java.lang.Class API
+ * @modules java.base/java.lang:open
+ * @compile classFileVersions.jcod
+ * @run main/othervm --enable-preview ClassFileVersionTest
+ */
+
+import java.lang.reflect.*;
+
+public class ClassFileVersionTest {
+
+    static Method m;
+
+    public static void testIt(String className, int expectedResult) throws Exception {
+        Class<?> testClass = Class.forName(className);
+        int ver = (int)m.invoke(testClass);
+        if (ver != expectedResult) {
+            throw new RuntimeException(
+                "expected " + expectedResult + ", got " + ver + " for class " + className);
```

Comment:
It would be clearer to show the expected and actual in minor:major format. That way if the test fails we can easily see which bit is wrong.

Reply:
Seems odd to display as major:minor when it is stored as minor:majpor

Reply:
I think it seems more natural to display the major version first, but I can change it if you prefer.

Reply:
Do humans read this or computers?  A human would want major:minor because that would make more sense.  Isn't this a test?  Just a drive by comment.

Reply:
@dholmes-ora Please see latest commit that changes the test to display minor:major.

Reply:
Latest change is good.

@coleenp this is for humans reading failed test output. If it reports "expected 0:1 but got 1:1" then the natural tendency would be to look for an issue in the upper word, when in fact the problem is in the lower word.